src/lib/containers/Login.tsx

```diff
@@ -131,8 +133,8 @@ class Login extends React.Component<Props, State> {
     event.preventDefault()
     const redirectUrl = this.props.googleRedirectUrl
       ? this.props.googleRedirectUrl
-      : `${SynapseClient.getRootURL()}?provider=${SynapseClient.AUTH_PROVIDER}`
-    SynapseClient.oAuthUrlRequest(SynapseClient.AUTH_PROVIDER, redirectUrl)
+      : `${SynapseClient.getRootURL()}?provider=${GOOGLE_AUTH_PROVIDER}`
```

Comment:
Login component still only supports Google SSO login for now.  We can add other providers (with their specific UI) later, as they become available.